src/plugins/yamlcpp/README.md

```diff
@@ -3,7 +3,7 @@
 - infos/licence = BSD
 - infos/needs =
 - infos/provides = storage/yaml
-- infos/recommends =
+- infos/recommends = base666
```

Comment:
or require? What happens with binary data without base666? I think the user does not expect to receive base64 encoded strings? (Does the standard says something about it?)

Reply:
> What happens with binary data without base666?

Then the user receives a base64 encoded string.

> I think the user does not expect to receive base64 encoded strings?

That depends, if he read the documentation of the plugin, then he would probably know that YAML CPP does not support base64 encoded binary data fully without the Base 666 plugin.

> Does the standard says something about it?

As far as I can remember base64 support is not required. Otherwise [yaml-cpp](https://github.com/jbeder/yaml-cpp) would probably save the data in encoded form.